x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_exploration/components/regression_exploration/regression_exploration.tsx

```diff
@@ -45,9 +58,42 @@ export const RegressionExploration: FC<Props> = ({ jobId, jobStatus }) => {
         setJobConfig(analyticsConfigs.data_frame_analytics[0]);
         setIsLoadingJobConfig(false);
       }
-    })();
+    } catch (e) {
+      if (e.message !== undefined) {
+        setJobConfigErrorMessage(e.message);
+      } else {
+        setJobConfigErrorMessage(JSON.stringify(e));
+      }
```

Comment:
I've seen this logic in a couple of places already, maybe we should create a function (or custom hook) to handle it?

Reply:
For now I think since this logic is fairly short and clear it's okay as is. 👍